src/defs.js

```diff
@@ -297,6 +297,12 @@ if (WEB) {
     const LUA_CPATH_DEFAULT =
         LUA_CDIR + "?.js;" + LUA_CDIR + "loadall.js;./?.js";
     module.exports.LUA_CPATH_DEFAULT = LUA_CPATH_DEFAULT;
+
+    const LUA_WEB_PATH_DEFAULT = "?.lua;?/init.lua";
```

Comment:
We shouldn't need new variables for this. `if(WEB)` should mean that `LUA_CPATH_DEFAULT` is already this.